gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -587,60 +628,86 @@ def add_variant_annotations(
     """
```

Comment:
It looks like this function is only used once in this script and is a lot more simplified than when you originally made it. If it isn't used in another file, the I think you can just get rid of the function and move the row annotation to where the function is used, and add the description as a comment or logger